lib/oxidized/model/uplinkolt.rb

```diff
@@ -0,0 +1,55 @@
+class UPLINKOLT < Oxidized::Model
+
+    prompt /^([\w.@()-]+[#>]\s?)$/
+    comment  '! '
+  
+    cmd :all do |cfg|
+      cfg.gsub! /^% Invalid input detected at '\^' marker\.$|^\s+\^$/, ''
+      cfg.gsub!(/^show running-config$/, '')
+      cfg.gsub!(/^.*\s*#\s*$/, '')
+  
+      # Remove leading and trailing whitespace
+      cfg.strip!
+  
+      # Remove empty lines
+      cfg.gsub!(/^\s*$/, '')
+  
+      cfg
+    end
+      
+    cmd 'configure terminal' do
+      # Enter configure terminal mode
+      cmd 'show version' do |cfg|
+      cfg.gsub! /^show version/, ''
+      comment cfg
+      end
+      
+    end
+    cmd 'show time' do |cfg|
```

Comment:
why? This way your config changes every time oxidized pulls?

Reply:
well it was not in my initial code. Then i saw it somewhere while going through other models, so putted it. 
Was having the same confusion that was it necessary or not. Now as you have pointed out I am removing it.